Dockerfile_pypi

```diff
@@ -5,9 +5,14 @@ WORKDIR app
 RUN apt-get update && apt-get autoclean && \
     apt-get install -y -q software-properties-common gnupg
 
+# Install ca-certificates
+RUN apt-get install -y ca-certificates
+COPY ./ca-certificates* /app/ca-certificates
+RUN cp /app/ca-certificates/*.crt /usr/local/share/ca-certificates | true && update-ca-certificates
+
 RUN  apt-get install -y -q --allow-unauthenticated git python3-pip
 
-RUN pip3 --version && pip3 install secureli
+RUN pip3 --version && pip3 install secureli --break-system-packages
```

Comment:
what does this do? --break-system-packages?

Reply:
without that flag, installing secureli on the latest debian image fails because we are not using a virtual python environment.  In this case, that's fine because these are ephemeral docker images used to run tests. Unless the purpose of this Dockerfile is to test the installation of securli through pip and thus, we should update it to use a virtual env.